platform/authorized_api_guide.md

```diff
@@ -38,22 +81,255 @@ curl 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
+         "privacyCookieDomainsEnabled":false,
+         "privacyCookieExpirationPeriod":31536000,
+         "privacyCookieDomains":[
+
+         ],
+         "organization":"default",
+         "appType":"web",
+         "gdprLocationRecognition":false
+      }
+   }
+}
+
+```
+#### Get an app
+
+Now, when app is added, it is possible to get it.
+
+Request example:
+```
+GET /api/apps/v2/<appId>
+```
+
+```
+curl 'https://testing.piwik.pro/api/apps/v2/b30e538d-4b05-4a75-ae25-7eb565901f38' -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json"
+```
+
+
+>Notice: URL contains `b30e538d-4b05-4a75-ae25-7eb565901f38`. What is it? It is unique ID of an app. 
+If you want to update given resource you must specify which one. How to obtain this ID?
+You can obtain ID from response's 'data/id' field when you added an app
+
+Response example:
+```
+{
+   "data":{
+      "type":"ppms\/app",
+      "id":"b30e538d-4b05-4a75-ae25-7eb565901f38",
+      "attributes":{
+         "name":"AppName",
+         "addedAt":"2018-09-13T12:16:30+00:00",
+         "urls":[
+            "http:\/\/example.com"
+         ],
+         "timezone":"UTC",
+         "currency":"USD",
+         "excludeUnknownUrls":false,
+         "keepUrlFragment":true,
+         "eCommerceTracking":false,
+         "siteSearchTracking":true,
+         "siteSearchQueryParams":[
+            "q",
+            "query",
+            "s",
+            "search",
+            "searchword",
+            "keyword"
+         ],
+         "siteSearchCategoryParams":[
+
+         ],
+         "delay":500,
+         "excludedIps":[
+
+         ],
+         "excludedUrlParams":[
+
+         ],
+         "excludedUserAgents":[
+
+         ],
+         "gdpr":true,
+         "gdprUserModeEnabled":false,
+         "privacyCookieDomainsEnabled":false,
+         "privacyCookieExpirationPeriod":31536000,
+         "privacyCookieDomains":[
+
+         ],
+         "organization":"default",
+         "appType":"web",
+         "gdprLocationRecognition":false
+      }
+   }
+}
+```
+ 
+#### Update app
+
+Consider you added app, but afterwards you want to change its name.
+
+Request example:
+```
+PATCH /api/apps/v2/<appId>
+```
+
+```
+curl -X PATCH 'https://<domain>/api/apps/v2/b30e538d-4b05-4a75-ae25-7eb565901f38' -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json" -v --data '{
+  "data": {
+    "attributes": {
+      "name": "NewAppName"
+    },
+    "type": "ppms/app",
+    "id": "b30e538d-4b05-4a75-ae25-7eb565901f38"
+  }
+}'
+```
+
+This request changed app name from `AppName` to `NewAppName`. 
+> Notice three things:
+> * `-X PATCH` before URL. It means that this request is available using `HTTP PATCH method`
+> * you have to specify also `data/id` - it's a [JSON API](http://jsonapi.org/) requirement
+> * also `data/type` is important. For example, when you want to work with app resource, specify it's type as `ppms/app` 
```

Comment:
`is important and also required`?

Reply:
I left only `required`. It should be clear that this way it is important